.circleci/config.yml

```diff
@@ -133,19 +133,20 @@ jobs:
             # exit early if any test fails (-failfast only works within a single
             # package).
             exit_code=0
-            pkgs=$(go list ./... | circleci tests split)
-            echo "Running $pkgs"
+            pkgs=$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)
+            echo "Running $(echo $pkgs | wc -w) packages:"
+            echo $pkg
```

Comment:
this is a small optimization. Mostly it doesn't improve the total test runtime by that much